### PR TITLE
AC-665: Create phpcs static check for ConfigTest

### PR DESCRIPTION
**Pull request by svera**
This sniff will check `config.xml` files looking for obsolete nodes occurrences in them, using `Magento2/Sniffs/Legacy/_files/obsolete_config_nodes.php` as reference.

### CODE REVIEW

File: Magento2/ruleset.xml [hunk 1]
```diff
@@ -115,6 +115,13 @@
         <severity>10</severity>
         <type>error</type>
     </rule>
+    <rule ref="Magento2.Legacy.ObsoleteConfigNodes">
```
Comment (jcuerdo) [inline Magento2/ruleset.xml:118]: I think the correct name is "ObsoleteNodeInConfig"

**Review by jcuerdo — CHANGES_REQUESTED** [review]

File: Magento2/ruleset.xml (hunk 1 above)
Comment (svera) [reply Magento2/ruleset.xml:118]: @jcuerdo this is referring a to nodes specific to config, so it's also correct to call them this way. Also, old rule is called also config nodes.

**Review by eliseacornejo — APPROVED** [review]

**Review by jcuerdo — APPROVED** [review]

**Review by jcuerdo — APPROVED** [review]

**Comment by svera** [comment]
@magento import pr to magento-commerce/magento-coding-standard

**Comment by m2-github-services** [comment]
@svera the Pull Request is successfully imported.

**Pull request merged**